docs/Recursion/catalannumber.md

```diff
@@ -0,0 +1,102 @@
+---
+id: Catalan Number 
+title: Catalan Number Using Recursion
+sidebar_label: Generate Catalan Number 
+description: "The Catalan number sequence is a fundamental concept in combinatorics, appearing in numerous counting problems such as valid parentheses expressions, binary search trees, and polygon triangulations. The nth Catalan number is defined recursively as:
+C(n)= i=0∑n−1
+       C(i)×C(n−i−1) with 𝐶(0)=1 . This recursive structure allows us to break complex problems into smaller subproblems, making Catalan numbers a versatile tool in algorithm design and mathematical theory. This contribution focuses on implementing the recursive approach for generating Catalan numbers."
```

Comment:
Your description does not follow good practices, and the equation is not allowed. Please carefully read our contribution guidelines.